public/i18n.js

```diff
@@ -1,84 +1,234 @@
+    "ftm-index-need-helpers": "We need to help the helpers, right now.",
+    "ftm-index-need-find-masks": "We need to find the masks",
+    "ftm-index-where-to-find-ppe": "Look in your basement, your emergency kit, your bug out bag, your workbench for N95 masks.If you find surgical masks, those are also in short supply, and in many places are being used in place of N95s. All of these masks can save lives now if you get them into the hands of health care workers.",
+    "ftm-index-types-of-ppe": "Other types of PPE like disposable <span class=embold>booties</span>, <span class=embold>safety goggles</span>, and <span class=embold>disposable suits</span> are also in short supply. If you have these things or know someone who might (HVAC installers? Construction workers?) please share this site with them. Large donors can register at <a href=\"https://getusppe.org/\">getusppe.org/give</a>.",
+    "ftm-what-to-do-header": "Here's what you can do:",
+    "ftm-what-to-do-1": "Look at our list of sites below. You can filter by state or supplies needed.",
+    "ftm-what-to-do-2": "If there's a site near you, drop off following their instructions.",
+    "ftm-what-to-do-3": "If you are donating open packages, please put them in a ziploc bag with a note saying whether they have been used, whether anyone in your household has been sick or travelled outside the country, and the expiration date if known.",
+    "ftm-what-to-do-4": "If there's not a site near you, visit <a href=\"https://getusppe.org/give\">getusppe.org</a> to let them know what you have available.",
+    "ftm-add-donation-site": "To add a donation site, update information, or request we take down information, please <a href=\"https://docs.google.com/forms/d/e/1FAIpQLSfgCpK5coPVFC6rJrE7ZhimiZuDoEaL6fo6gYqxsN_FIpJZhg/viewform\">fill in this form</a>. Your submission will be approved and updated as soon as possible.",
+    "ftm-donations-site-loading": "Loading donation sites...",
+    "ftm-filters-header": "Filters",
+    "ftm-project-leads": "Our leads are:",
+    "ftm-site-launch-info": "This site was launched on <a href=\"https://twitter.com/arrkay/status/1240704610186092546\">March 19th</a> by <a href=\"https://www.linkedin.com/in/rachelpopkin/\">Rachel Popkin</a> and has been improved by many fantastic volunteers. Many of us are tech workers in Seattle. We are advised by a group of doctors and nurses around the country.",
+    "ftm-anonymous-volunteers": "and others who are anonymous",
+    "ftm-ack-data-sources": "This site also includes data from <a href=\"https://tinyurl.com/hospitals-needing-ppe\">https://tinyurl.com/hospitals-needing-ppe</a>, a project by:",
+    "ftm-project-volunteers": "Our volunteers include:"
+  },
+  "fr-FR": {
```

Comment:
just pseudo translations right now for testing, will copy in translations once this works